types/node/v20/globals.d.ts

```diff
@@ -409,4 +410,8 @@ declare global {
         File: infer T;
     } ? T
         : typeof import("node:buffer").File;
+
+    interface WebSocket extends _WebSocket {}
+    var WebSocket: typeof globalThis extends { onmessage: any; WebSocket: infer T } ? T
+        : typeof import("undici-types").WebSocket;
```

Comment:
This is not enabled in v20 by default.

Reply:
Yes, this is enabled via a CLI flag. What should we do in this case?

Reply:
I see that we have defined `vm.Module` which only available with flag too.

https://nodejs.org/docs/latest-v20.x/api/vm.html#class-vmmodule

Reply:
I would recommend to not type this in v20.

Reply:
Ok, I removed types for v20